spec/features/orders/00_user_can_see_orders_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+  RSpec.feature "an existing user has multiple orders" do
+    context "as an authenticated user" do
+      scenario "I see only my orders" do
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(logged_user)
+        item1 = Item.create(matcha)
+        item2 = Item.create(chopsticks)
+        item3 = Item.create(fish)
+
+        Cart.new(item1.id.to_s => 1)
```

Comment:
Do we need to make all these carts? I feel like we should be adding the items to the cart instead of creating a cart for each item.